vagrant_bash_3/hw9_vagrant_provision.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash -e
+
+USER_NAME="hw9"
+HOME_DIR="/opt"
+ETC_HOSTS="/etc/hosts"
+ETC_HOSTS_RECORD="127.0.0.1 myownapp.com"
+VM_NAME="vagrant-ubuntu-trusty-64"
+PASSWORD_AUTH="PasswordAuthentication yes"
+PASSWORD_NO_AUTH="PasswordAuthentication no"
+SSHD_CONF_PATH="/etc/ssh/sshd_config"
+ROOT_LOGIN_ON="PermitRootLogin without-password"
+ROOT_LOGIN_OFF="PermitRootLogin no"
+PACKAGES="mc vim git"
+RESOLV_CONF="/etc/resolv.conf"
+DNS_1="8.8.8.8"
+DNS_2="1.1.1.1"
+MY_APP="myownapp.com"
+CREATION_DIR_PATH="/var/log/myownapp"
+DEFAULT_GROUP="root"
+
+function add_user {
+  echo "Start create user: $USER_NAME"
+  sudo useradd -s /bin/bash -d ${HOME_DIR}/${USER_NAME} -m -G wheel ${USER_NAME}
+}
+
+function ssh_key {
+  echo "Generate ssh keys for: $USER_NAME"
+  sudo mkdir ${HOME_DIR}/${USER_NAME}/.ssh
+  sudo chmod 700 ${HOME_DIR}/${USER_NAME}/.ssh
+  sudo su $USER_NAME
+  ssh-keygen -t rsa -f ${HOME_DIR}/${USER_NAME}/.ssh/id_rsa -q -P "" -C "${USER_NAME}@${VM_NAME}"
```

Comment:
should be an existing key to be able to log in.

Reply:
use existing key